minecraft-server/minecraftd.sh

```diff
@@ -118,9 +117,9 @@ idle_server_daemon() {
 		# Retry in ${CHECK_PLAYER_TIME} seconds
 		sleep ${CHECK_PLAYER_TIME}
 
-		if screen -S "${SESSION_NAME}" -Q select . > /dev/null; then
+		if tmux -L "${SESSION_NAME}" select-window -t0:0 2> /dev/null; then
```

Comment:
I think `has-session` session instead of `select-window` should be sufficient. We don't need to check for a specific target since everything is separated by socket-name, right?

Reply:
That's fair. Initially I though using select-window to check the session also helps to be sure the commands are sent to the correct window, but in reality I should just specify the panel in the send-key command itself.

Reply:
The question basically boils down to whether you can image the implementation to use multiple sessions and/or panes with the same socket at some point.

A potential use case might be to manage multiple servers with a single script. At least a user proposed that some time ago. One might imagine that having multiple sessions might be useful then. However, I think one should properly separate all minecraft instances e.g. via containerization or the like for added security.

If you think this might be useful at a later point I am also more than happy to keep the current approach!